dotcom-rendering/src/web/lib/decidePalette.ts

```diff
@@ -1084,6 +1088,7 @@ export const decidePalette = (format: ArticleFormat): Palette => {
 			headlineTag: backgroundHeadlineTag(format),
 			mostViewedTab: backgroundMostViewedTab(format),
 			matchNav: backgroundMatchNav(),
+			underline: backgroundUnderline(format),
```

Comment:
What do you think about `analysisUnderline`? Assuming that this colour is exclusively for analysis pieces, that might make it clearer